src/components/DefinitionEntry.js

```diff
@@ -71,16 +71,28 @@ export default class DefinitionEntry extends React.Component {
 
   renderHeadline(definition) {
     const { namespace, name, revision } = definition.coordinates
+    const locationUrl = this.getLocationURL(definition)
+    const revisionUrl = this.getRevisionUrl(definition)
     const namespaceText = namespace ? namespace + '/' : ''
     const sourceUrl = this.getSourceUrl(definition)
-    let revisionText = <span>&nbsp;&nbsp;&nbsp;{revision}</span>
+    let revisionText = (
+      <span>
+        &nbsp;&nbsp;&nbsp;
+        <a href={revisionUrl} target="_blank">
+          {revision}
+        </a>
+      </span>
+    )
+
     const location = get(definition, 'described.sourceLocation')
```

Comment:
note that `location` here is the **source** location, not the location of the component itself. We should likely just stop showing the source location in the two line view

Reply:
I have removed source location from the two line view.